docs/source-pytorch/accelerators/gpu_intermediate.rst

```diff
@@ -491,7 +491,7 @@ Below are the possible configurations we support.
 
 Implement Your Own Distributed (DDP) training
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-If you need your own way to init PyTorch DDP you can override :meth:`pytorch_lightning.strategies.ddp.DDPStrategy.init_dist_connection`.
+If you need your own way to init PyTorch DDP you can override :meth:`pytorch_lightning.strategies.ddp.DDPStrategy._init_dist_connection`.
```

Comment:
This method does not exist. We should probably remove this or update it

Reply:
updated